docs/proposals/kube_events_design_doc.md

```diff
@@ -0,0 +1,136 @@
+Collect and store kubernetes events
+===
+By kubernetes events we understand log messages internal to kubernetes, accessible through the kubernetes API `/api/v1/events?watch=true`, originally stored in etcd. The etcd storage has time and performance constraints, therefore, we would like to collect and store them permanently in EFK.
```

Comment:
@wozniakjan I point mentioned by @goern was stock event router does not collect Openshift events.  If this is still true we need to understand:
* What changes upstream need to be made
* Start work
* Carry a patch until we can get an upstream merge.

At a minimum there needs to be a statement expressly stating Openshift events are not included.  I also think it would be useful to add: https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#v1-event

Reply:
@jcantrill  fact that eventrouter collects only events from `/api/v1/events?watch=true` was mentioned in the original design doc as well from the very beginning. I was under the impression it is a known and accepted issue.

An entry point to figure out how many changes we will need in order to get every Openshift event to through the pipeline is here
https://github.com/heptio/eventrouter/blob/master/main.go#L110

I think we will need to write our own informers for each Openshift API that releases any events.